example/src/App.tsx

```diff
@@ -63,20 +63,17 @@ export default function App() {
     listCamerasAsync();
   }, [permissionsGranted]);
 
+  /* eslint-disable react-hooks/exhaustive-deps */
   useEffect(() => {
     if (!permissionsGranted) {
       return;
     }
 
-    let localVideoTrackResolve: (
-      value?: LocalVideoTrack | PromiseLike<LocalVideoTrack> | undefined
-    ) => void;
-    const localVideoTrackPromise = new Promise<LocalVideoTrack>(
-      (resolve) => (localVideoTrackResolve = resolve)
-    );
-
     const asyncCreateLocalVideoTrack = async () => {
       try {
+        if (localVideoTrack) {
+          await localVideoTrack.destroy();
```

Comment:
this sucks, but I can't think of a way to fit it into a cleanup hook since it's async and must complete before we start the subsequent `create`

Reply:
yea i think this is really the only option given what you were saying yesterday about how it seems that we need to pipeline / sequence these video track operations.

Reply:
so, what ends up happening when we unmount this component. does the underlying videoTrack get left around in androidland?

Reply:
Oh good point, yeah I think so.

I think I want to make a more involved hook for this